src/poker.js

```diff
@@ -1,5 +1,19 @@
 class Poker {
-  // Write your implementation here
+  winningPair(player1, player2) {
+    if (player1[0] !== player1[1] && player2[0] !== player2[1]) {
+      return [];
```

Comment:
I tried removing the if else statements in my code and used only if , which worked.